src/utils/dom/createNode.js

```diff
@@ -21,7 +21,14 @@ export default function createNode(
   const result = doc.createElement(tag);
 
   Object.keys(attrs).forEach(key => {
-    result.setAttribute(key, attrs[key]);
+    const value = attrs[key];
+
+    if (typeof value === "function") {
```

Comment:
Hey @wgamboa, I just saw this. I'm not sure how I feel about it, since if the value is a function, then it technically isn't an attribute, but a property instead. I almost wonder if we should rename `attrs` to `props` and apply all the key-value pairs using `result[key] = value`.

Reply:
@Aaronius you can change it in your next pr if you want.